datasets/esa-worldcover.yaml

```diff
@@ -21,6 +21,7 @@ Tags:
   - disaster response
   - mapping
   - synthetic aperture radar
+  - land cover
```

Comment:
@dzanaga can you add the same tags from https://registry.opendata.aws/esa-worldcover/?